service/src/main/resources/schema/development/schema.json

```diff
@@ -180,6 +191,19 @@
           "type": "integer"
         }
       }
+    },
+    "samples": {
+      "type": "object",
+      "properties": {
+        "disease": {
+          "type": "array",
+          "items": { "type": "string" }
+        },
+        "species": {
+          "type": "array",
```

Comment:
For this, we should use https://www.ebi.ac.uk/ols/ontologies/ncit/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNCIT_C14250&lang=en&viewMode=All&siblings=false

Maybe we should add a link to this to the description, not sure how best to add it to the json schema